web/html/src/manager/virtualization/nets/properties/Vlans.js

```diff
@@ -0,0 +1,78 @@
+// @flow
+import * as React from 'react';
+import Validation from 'components/validation';
+import { FormContext } from 'components/input/Form';
+import { FormMultiInput } from 'components/input/FormMultiInput';
+import { Text } from 'components/input/Text';
+import { Select } from 'components/input/Select';
+
+type Props = {
+};
+
+export function Vlans(props: Props) {
+  const formContext = React.useContext(FormContext);
+  const model = formContext.model || {};
+  return(
+    <div className="col-md-12">
+      <div className="col-md-7 col-md-offset-2">
+        <FormMultiInput
+          id="vlans"
+          title={t('VLANs')}
+          prefix={"vlans"}
+          onAdd={(index) => {
+            formContext.setModelValue(`vlans${index}_tag`, '');
+            if (index > 0) {
+              formContext.setModelValue(`vlantrunk`, true);
+            }
+          }}
+          onRemove={(index: number) => {
+            Object.keys(model)
+              .filter(key => key.startsWith(`vlans${index}_`))
+              .forEach(key => formContext.setModelValue(key, undefined));
+            if (Object.keys(model).filter(key => key.match(/^vlan[0-9]+_tag$/)).length <= 1) {
```

Comment:
This line could use a comment describing what the logic here is, without any context it's a bit confusing why we want to do this on both 0 and 1 matches for example.

Reply:
ACK